### PR TITLE
add S1 

### PR DESCRIPTION
**Pull request by ferrighi**
As for #63 : 
1) add sentinel-1 mmd example file. 
2) Adding ODATA in list of valid access entries
3) make last_metadata_update/update/note not mandatory

Main changes with previous mmd are in last_metadata_update, platform, datacenter contact (moved to personnel). I have not included cloud and scene coverage, but maybe timeliness apply. Please @hevgyrt just add it if needed.

### CODE REVIEW

File: input-examples/sentinel-1.mmd [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        <orbit_absolute>32205</orbit_absolute>
+        <orbit_direction>descending</orbit_direction>
+        <instrument>
+                <short_name>SAR-C</short_name>
+                <long_name>Synthetic Aperture Radar (C-band)</long_name>
+                <resource>https://www.wmo-sat.info/oscar/instruments/view/450</resource>
+                <mode>EW</mode>
+                <polarisation>HH+HV</polarisation>
+                <product_type>GRD</product_type>
+        </instrument>
```
Comment (hevgyrt) [inline input-examples/sentinel-1.mmd:65]:
```suggestion
        </instrument>
        <ancillary>
                <timeliness>NRT</timeliness>
        </ancillary>
```

Comment (hevgyrt) [reply input-examples/sentinel-1.mmd:65]: This is relevant for these Sentinel-1 products

**Comment by ferrighi** [comment]
1. I think you are right. Maybe we should change file.mmd into file-mmd.xml
2. @steingod , should we use the namespace mmd in the elements? It is not so in the documentation.

**Comment by steingod** [comment]
Concerning 1. - for sure, concerning 2. - it is not needed, it is more a pedagogical question. I think it is good to continue using mmd: prefix (I do so in most files). Speaking of which has  gone under my radar, usage of xml: prefix for language. I think this is good practise, but would like to check if we should use mmd: instead.

**Comment by ferrighi** [comment]
Excellent. I will fix these two things and push changes, then @hevgyrt you can merge when ready.

**Pull request merged**